src/apps/FEMSWEEP.cpp

```diff
@@ -98,6 +99,67 @@ void FEMSWEEP::setUp(VariantID vid, size_t RAJAPERF_UNUSED_ARG(tune_idx))
   allocAndInitDataRandValue (m_M0dat    , m_M0len     , vid);
   allocAndInitDataRandValue (m_Xdat     , m_Xlen      , vid);
 
+  // Read mesh connectivity data from file.
```

Comment:
This should probably use allocForInit instead of copying the data through an extra buffer. See an example here. https://github.com/LLNL/RAJAPerf/blob/06fc079971f6491f1ed9cee3369ef214527bb84f/src/algorithm/HISTOGRAM.cpp#L57